functions/common.php

```diff
@@ -168,3 +167,18 @@ function gnt_force_redirect($url) {
         </script><?php
     }
 }
+
+/**
+ * Hide Row if setting is disabled
+ * @param  string $setting The setting to check
+ * @echo string 'display:none;' if setting is disabled
+ */
+ function gnt_hide_row($setting) {
```

Comment:
It might be good to rename the function `gnt_should_hide_row_e` since the function echoes something and does not return it.  This is similar to the `__()` and `_e()` functions.